src/rviz_visual_tools.cpp

```diff
@@ -1077,6 +1077,47 @@ bool RvizVisualTools::publishPolygon(const geometry_msgs::Polygon &polygon, cons
   publishPath(points, color, scale, ns);
 }
 
+  bool RvizVisualTools::publishWireframeCuboid(const Eigen::Vector3d &position,
+                                               const Eigen::Matrix3d &rotation_matrix,
+                                               const Eigen::Vector3d &min_point,
+                                               const Eigen::Vector3d &max_point,
+                                               const RvizVisualTools::colors color=RvizVisualTools::BLUE)
```

Comment:
Instead of specifying at position and rotation matrix, it should just pass in an ``Eigen::Affine3d`` that contains both, as is standard in all the other functions in this class.